schemas/+behavior/TowersSession.m

```diff
@@ -52,7 +55,11 @@ function insertTowersSessionFromFile(self, key,  log)
             % log  = behavioral file as stored in Virmen
             
             %Write stimulus_set
-            key.stimulus_set = log.animal.stimulusSet;
+            if isstruct(log.animal) && isfield(log.animal, 'stimulusSet') && ~isnan(log.animal.stimulusSet)
+                key.stimulus_set = log.animal.stimulusSet;
+            else
+                key.stimulus_set = -1;
```

Comment:
what does -1 mean? shall we document in the table def?

Reply:
In this case, -1 means stimulus_set was not present.
I guess the most correct thing would be to set it to NULL. 
I set it to -1 because this field is not nullable in the database.
We can change the field to be nullable.